Framework/Algorithms/src/GenerateEventsFilter.cpp

```diff
@@ -1168,8 +1166,7 @@ void GenerateEventsFilter::makeMultipleFiltersByValuesPartialLog(
   const Types::Core::DateAndTime ZeroTime(0);
```

Comment:
Can you also remove this one to fix the remaining `unreadVariable` in this section, or does this create other issues?

Reply:
I think you meant `int currindex = -1;` on line 1168. I removed the assignment.